docs/content/getting_started/centos.md

```diff
@@ -0,0 +1,28 @@
+# CentOS
+
+{!content/getting_started/minimum_requirements.md!}
+
+### Installation Instructions
+#### Pre-Reqs
```

Comment:
We should discuss headings for all of your pages, why is this fourth level? Only 2nd level appears on the left-hand side table-of-contents, so right now it is mostly empty which doesn't seem correct.

```markdown
#CentOS

## Minimum Requirements

## Pre-Reqs

## Clone MOOSE

## Compile libMesh

## blah, blah, blah
```